mythril/analysis/security.py

```diff
@@ -45,42 +51,57 @@ def get_detection_module_hooks(modules, hook_type="pre"):
     return dict(hook_dict)
 
 
-def get_detection_modules(entrypoint, include_modules=()):
+def get_detection_modules(entrypoint, include_modules=(), custom_modules_directory=""):
```

Comment:
This method could be more concise by first checking for custom modules, then creating a list of packages with something like this:
```python
custom_packages = list(pkgutil.walk_packages(custom_modules_directory)) if custom_modules_directory else []`
packages = list(pkgutil.walk_packages(modules.__path__)) + custom_packages

for loader, module_name, _ in packages:
    ...
```